tests/rancher/rancher_k3s_master.pm

```diff
@@ -0,0 +1,60 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: One master node whene k3s server runs and cluster is tested..
+# Maintainer: Pavel Dostal <pdostal@suse.com>
+
+use base 'x11test';
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use utils;
+use mm_network;
+use rancher::utils;
+use Utils::Systemd 'disable_and_stop_service';
+
+sub run {
+    my ($self) = @_;
+    $self->select_serial_terminal;
+
+    systemctl('start sshd.service');
+    disable_and_stop_service('firewalld');
+
+    mutex_wait('support_server_ready');
+    configure_hostname(get_required_var('HOSTNAME'));
+    configure_dhcp();
+
+    script_retry("nslookup gitlab.com", delay => 3, retry => 10);
+
+    assert_script_run("curl -L https://get.k3s.io -o ~/get_k3s", 360);
+    assert_script_run("chmod +rx ~/get_k3s");
+
+    assert_script_run("./get_k3s", 600);
+    script_retry("kubectl get node | grep ' Ready '", delay => 3, retry => 10);
+    script_run("cat /var/lib/rancher/k3s/server/node-token");
```

Comment:
I think you need to add something like this `while [ ! -f /var/lib/rancher/k3s/server/node-token ]; do sleep 1; done`
so you can simply delete [this](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/12200/files#diff-f0a0d9751ff653064cde0597b22dcf55f893fec5d5d10daa05b1c47d347311faR41)

Reply:
Should be solved now.